e2e/tests/attendance-tests.spec.ts

```diff
@@ -103,33 +98,14 @@ test.describe("Attendance Module", () => {
     /*
       Verify the class just recorded attendance should be highlighted in green and all others in orange
     */
-    const BackOverviewBtn = page.getByRole("button", {
+    const backOverviewBtn = page.getByRole("button", {
       name: "Back to Overview",
     });
-    await expect(BackOverviewBtn).toBeVisible(); // "Back to Overview" button is visible
-    await BackOverviewBtn.click(); // "Back to Overview" button is clickable
-
-    const classCard = page.locator("mat-card");
-    const classCardCount = await classCard.count();
-    for (let i = 0; i < classCardCount; i++) {
-      const card = page.locator("mat-card").nth(i);
-
-      // Extract the border color
-      const borderColor = await card.evaluate(
-        (el) => getComputedStyle(el).borderLeftColor,
-      );
-
-      if (i === 0) {
-        expect(borderColor).toBe("rgb(76, 175, 80)"); // Green
-      } else {
-        expect(borderColor).toBe("rgb(255, 152, 0)"); // Orange
-      }
-    }
```

Comment:
did you intentionally remove the color checks?

(I guess a better way for this will be the visual regression testing soon ...)